netaddr.go

```diff
@@ -47,7 +45,7 @@ const o4 = len(mapped4Prefix)
 // Its memory representation is 24 bytes on 64-bit machines (the same
 // size as a Go slice header) for both IPv4 and IPv6 address.
 type IP struct {
-	a [16]byte // IPv6 or IPv4-mapped IPv6 form of IPv4 addr
+	hi, lo uint64 // IPv6 or IPv4-mapped IPv6 form of IPv4 addr
```

Comment:
Say something about endianness here. Maybe write out an example IPv6 address and indicate what the hi and lo values are for it (in hex, so that the correspondence is obvious).

Reply:
Done.